docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -150,24 +151,104 @@ On Level 1 systems, the commands in the ``build_<platform>_<compiler>.env`` file
 
 .. code-block::
 
-   source env/build_hera_intel.env
+   source $SRW/env/build_hera_intel.env
 
 from the main ``ufs-srweather-app`` directory to source the appropriate file.
 
-On Level 2-4 systems, users will need to modify certain environment variables, such as the path to NCEP libraries, so that the SRW App can find and load the appropriate modules. For systems with Lmod installed, one of the current ``build_<platform>_<compiler>.env`` files can be copied and used as a template. To check whether Lmod is installed, run ``echo $LMOD_PKG``, and see if it outputs a path to the Lmod package. On systems without Lmod, users can modify or set the required environment variables with the ``export`` or ``setenv`` commands despending on whether they are using a bash or csh/tcsh shell, respectively: 
+On Level 2-4 systems, users will need to modify certain environment variables in the ``build_<platform>_<compiler>.env`` file, such as the path to HPC-Stack, so that the SRW App can find and load the appropriate modules. For systems with Lmod installed, one of the current ``build_<platform>_<compiler>.env`` files can be copied and used as a template. To check whether Lmod is installed, run ``echo $LMOD_PKG``, and see if it outputs a path to the Lmod package. On systems without Lmod, users can modify or set the required environment variables with the ``export`` or ``setenv`` commands despending on whether they are using a bash or csh/tcsh shell, respectively: 
```

Comment:
@gspetro-NOAA - Some changes proposed for the first sentence in line 158:
"
On Level 2-4 systems, users will need to modify certain environment variables in the ``build_<platform>_<compiler>.env`` file, such as the path to HPC-Stack, so that the SRW App can find and load the software modules. A list of software modules built on a local system could be given directly in ``build_<platform>_<compiler>.env`` , or  provided in a standard ``srw_common`` module that is loaded.   
"

(please feel free to re-word it if needed!)

Reply:
I think I address this below in the MacOS-specific section. For Level 1 machines, there shouldn't be any need to touch the build_<platform>_<compiler>.env or srw_common files.